portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,51 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div class="topnav" id="myTopNav">
+      <a href="#home" class="active">Home</a>
+      <a href="#about">About me</a>
+      <a href="#project">Projects</a>
+      <a href="#hobbies">What I enjoy</a>
+      <a href="#gallery">Gallery</a> 
+      <a href="#blog">Blog Posts</a>
+    </div>
+    <div class="content">
+      <div class="cover" id="home">
+        <h>Annie Yang</h>
+      </div>
+      <div class="about" id="about">
+        <h1>About Me</h1>
+        <p>Hi! My name is Annie and I’m currently a second year Software Engineering student at UNSW and a STEP Intern at Google 2020. 
+            I’m into software development, especially in the backend side and security engineering. 
+            My passion for technology and software stemmed from my curiosity and obsession with video games when I was younger. 
+            I was always curious into how games were developed and really wanted to make one myself. 
+            That was what put me into a path of Software Engineering. 
+            Nowadays I still spend a lot of my free time playing video games, however I have shifted my career interest from games to general software programming.</p>
+        <p>In my spare time, besides from playing video games, I enjoy hiking, cycling and working out.</p>
+      </div>
+      <div id="project">
+        <h1>Projects</h1>
+        <p>Projects I have worked on in the past few years include:</p>
+        <ul>
+          <li>Card game bot</li>
+          <p>Created a bot in C which played the mathematical card game 'CoCo'.</p>
+          <li>Pokemon Pokedex</li>
+          <p> Created a Pokedex for Pokemon using doubly linked-lists in C.
```

Comment:
Nit: missing the closing <\/p>. Otherwise LGTM

Reply:
Fixed. Thanks!